_posts/2019-08-05-cognito-azuread-sso.markdown

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title:  "Securing your web applications in AWS using SSO with Azure Active Directory"
+date:   2019-08-05
+tags: aws cognito oidc saml2 sso azure
+author: Paul Symons
+image: 
```

Comment:
May need an image?

Reply:
will strip out this param

Reply:
it may insert a default / revolving image - better to have one